package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@whereby.com/browser-sdk",
-  "version": "2.0.0-beta.0",
+  "version": "2.0.0-alpha",
```

Comment:
Add `.0` after `alpha` so that we can iterate on the tag before moving to beta?

Reply:
We can still iterate with `alpha1` or `alpha.1`. It's not common to use `0` for `alpha` versons
https://semver.org/#spec-item-9

Reply:
Gotcha. I'm fine with that, just saw this `.x` syntax in some guides 👍